src/http/server.ts

```diff
@@ -303,6 +313,7 @@ export class Server {
         throw new Drash.Exceptions.HttpResponseException(
           418,
           "The response must be returned inside the " +
+            resource.constructor.name + "." +
             request.method.toUpperCase() + " method of the resource",
```

Comment:
This results in `The response must be returned inside the HomeResource.GET method of the resource`, wonder if it sounds better to you for it to be `The response must be returned inside the GET method of the "HomeResource"`?

Reply:
that makes more sense

Reply:
0aa19c1